lib/plugins/aws/deploy/index.js

```diff
@@ -104,16 +121,34 @@ class AwsDeploy {
       'aws:deploy:deploy:updateStack': async () => {
         if (this.serverless.service.provider.shouldNotDeploy) return;
         await this.updateStack();
+        log.notice(
+          `\n${style.noticeSymbol('✔')} Service deployed to stack ${this.serverless
+            .getProvider('aws')
+            .naming.getStackName()}\n`
+        );
       },
 
       // Deploy finalize inner lifecycle
       'aws:deploy:finalize:cleanup': async () => {
         if (this.serverless.service.provider.shouldNotDeploy) return;
+        progress.get('main').notice('Updating');
         await this.cleanupS3Bucket();
         if (this.options.package || this.serverless.service.package.path) {
           await this.serverless.pluginManager.spawn('aws:common:cleanupTempDir');
         }
       },
+
+      'error': () => {
+        const isDeployCommand =
+          this.serverless.processedInput.commands.length === 1 &&
+          this.serverless.processedInput.commands[0] === 'deploy';
+        if (isDeployCommand && !this.options.function) {
+          log.notice(); // In order to ensure extra new line before the error message
```

Comment:
Let's do `log.error()`. Thing is that if someone will use `SLS_LOG_LEVEL=warning` then this new line will become ineffective for below error log.

One issue with using `log.error()` was that log reporter still was alos level prefix on empty messages, but I've just fixed that with `log` version v6.2.0, so after it's bumped in `@serverless/utils` we'll be sure it's not the case